src/app/iteration/iteration.component.html

```diff
@@ -179,12 +179,6 @@ <h3 class="flex-1">Iterations</h3>
     </div>
   </div>
 </div>
-<div *ngIf="loggedIn"
-  class="create-iteration-bar pointer"
-  (click)="modal.openCreateUpdateModal('create');">
-  <span><i class="pficon-add-circle-o fa-icon"></i>
```

Comment:
@mindreeper2420  you can make few changes in following files 
In file  :  iteration.component.html 
<i **id="add-iteration-icon"** class="fa fa-plus-circle fa-lg pull-right"></i>
In file : ..src/tests/work-item/work-item-list/page-objects/work-item-list.page.js
Make changes  
iterationAddButton  (){
    return element(by.id('add-iteration-icon'));
  }